apps/studio/src/pages/home/wallet/Wallet.tsx

```diff
@@ -82,8 +91,10 @@ const Wallet: FunctionComponent = () => {
           ) }
         </Box>
 
-        { !!unclaimedRoyalties && (
+        { unclaimedRoyalties ? (
           <UnclaimedRoyalties unclaimedRoyalties={ unclaimedRoyalties } />
+        ) : (
+          <NoPendingEarnings />
         ) }
```

Comment:
The actual conditional, per the Figma, should probably be if no songs have been uploaded (minted? might need to verify with Product?) -> then don't show either box  -> otherwise if song has been uploaded -> then unclaimedRoyalties conditional.

 Can be added to either of your PRs or I can modify on my STUD-95 PR.